packages/union-component/src/components/DonationFundDropdown.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { SelectHTMLAttributes, useEffect, useState } from 'react';
+import DonationDropdown from './DonationDropdown';
+import { getMembershipFunds, IFund } from './utils';
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
+  setFundId: (fundId: string) => void;
+}
+
+const DonationFundDropdown: React.FC<Props> = (props) => {
+  const { setFundId } = props;
+  const [options, setOptions] = useState<IFund[]>([]);
+
+  useEffect(() => {
+    const getOptions = async () => {
+      const funds = await getMembershipFunds();
+
+      if (funds?.length) {
```

Comment:
Do we have a sort of feedback if the response fails?
_Like what would happen if this short circuit remains falsy?_

Reply:
@duranmla I could add an error message to give some feedback to the user, I will check with Thiago